{"schemaVersion":{"patch":0,"minor":3,"major":0},"hierarchy":{"paths":[["doc:\/\/com.auth0.Auth0\/documentation\/Auth0","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError"]]},"sections":[],"variants":[{"paths":["\/documentation\/auth0\/credentialsmanagererror\/nocredentials"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"fragments":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"noCredentials"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"CredentialsManagerError","preciseIdentifier":"s:5Auth023CredentialsManagerErrorV"}],"symbolKind":"property","roleHeading":"Type Property","title":"noCredentials","externalID":"s:5Auth023CredentialsManagerErrorV02noB0ACvpZ","modules":[{"name":"Auth0"}],"role":"symbol"},"kind":"symbol","abstract":[{"text":"No credentials were found in the store.","type":"text"},{"text":" ","type":"text"},{"type":"text","text":"This error does not include a "},{"type":"reference","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/Auth0Error\/cause-9wuyi","isActive":true},{"type":"text","text":"."}],"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"tokens":[{"text":"static","kind":"keyword"},{"kind":"text","text":" "},{"kind":"keyword","text":"let"},{"text":" ","kind":"text"},{"text":"noCredentials","kind":"identifier"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError","text":"CredentialsManagerError","preciseIdentifier":"s:5Auth023CredentialsManagerErrorV"}],"platforms":["iOS"]}]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError\/noCredentials"},"references":{"doc://com.auth0.Auth0/documentation/Auth0/Auth0Error/cause-9wuyi":{"abstract":[{"type":"text","text":"The underlying "},{"code":"Error","type":"codeVoice"},{"type":"text","text":" value, if any. Defaults to "},{"type":"codeVoice","code":"nil"},{"text":".","type":"text"}],"kind":"symbol","url":"\/documentation\/auth0\/auth0error\/cause-9wuyi","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"cause","kind":"identifier"},{"kind":"text","text":": "},{"text":"Error","preciseIdentifier":"s:s5ErrorP","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"type":"topic","title":"cause","role":"symbol","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/Auth0Error\/cause-9wuyi"},"doc://com.auth0.Auth0/documentation/Auth0":{"type":"topic","kind":"symbol","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0","url":"\/documentation\/auth0","title":"Auth0","abstract":[{"type":"text","text":"SDK for Apple platforms."}],"role":"collection"},"doc://com.auth0.Auth0/documentation/Auth0/CredentialsManagerError/noCredentials":{"abstract":[{"type":"text","text":"No credentials were found in the store."},{"text":" ","type":"text"},{"text":"This error does not include a ","type":"text"},{"isActive":true,"identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/Auth0Error\/cause-9wuyi","type":"reference"},{"text":".","type":"text"}],"fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"noCredentials"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:5Auth023CredentialsManagerErrorV","text":"CredentialsManagerError"}],"role":"symbol","type":"topic","kind":"symbol","title":"noCredentials","url":"\/documentation\/auth0\/credentialsmanagererror\/nocredentials","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError\/noCredentials"},"doc://com.auth0.Auth0/documentation/Auth0/CredentialsManagerError":{"navigatorTitle":[{"text":"CredentialsManagerError","kind":"identifier"}],"url":"\/documentation\/auth0\/credentialsmanagererror","role":"symbol","kind":"symbol","type":"topic","abstract":[{"text":"Represents an error during a Credentials Manager operation.","type":"text"}],"identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError","title":"CredentialsManagerError","fragments":[{"kind":"keyword","text":"struct"},{"text":" ","kind":"text"},{"text":"CredentialsManagerError","kind":"identifier"}]}}}